contributor_docs/es/steward_guidelines.md

```diff
@@ -0,0 +1,381 @@
+# Directrices para Administradores  
```

Comment:
Traducir "stewards" como "supervisores"

Reply:
remplazado, excepto por el admin mencionado tal cual en dependabot:
Dependabot PRs are usually only visible to "repo admins " so if this does not apply to you, please skip this section.